.github/workflows/automatic-update.yml

```diff
@@ -40,7 +40,8 @@ jobs:
         with:
           token: ${{ steps.generate_token.outputs.token }}
           branch: new-update-plugin-examples
-          author: github-actions[bot] <41898282+github-actions[bot]@users.noreply.github.com>
+          #  Author field is ignored when commit-message is set to true
```

Comment:
>commit-message is set to true

Should it be when `commit-message is set` or `commit-message is not empty'?

Reply:
it's neither or, it's actually:

> when sign-commits is set to true

I don't know why I wrote commit-message before... fixed via https://github.com/grafana/grafana-plugin-examples/pull/506/commits/7d27e004aed1e151b387745768e317c03e7596a1